### PR TITLE
gnomeExtensions.easyScreenCast: init at unstable-2020-11-25

### PR DESCRIPTION
**Pull request by doronbehar**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/gnome-3/extensions/EasyScreenCast/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  ];
+
+  makeFlags = [ "INSTALLBASE=$(out)/share/gnome-shell/extensions" ];
+
+  uuid = "EasyScreenCast@iacopodeenosee.gmail.com";
+
+  meta = with stdenv.lib; {
+    description = "Simplifies the use of the video recording function integrated in gnome shell";
+    homepage = "https://github.com/EasyScreenCast/EasyScreenCast";
+    license = licenses.gpl2;
```
Comment (jtojnar) [inline pkgs/desktops/gnome-3/extensions/EasyScreenCast/default.nix:33]: `gpl2` is unclear, use `gpl2Plus` or `gpl2Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -26067,6 +26067,7 @@ in
     draw-on-your-screen = callPackage ../desktops/gnome-3/extensions/draw-on-your-screen { };
     drop-down-terminal = callPackage ../desktops/gnome-3/extensions/drop-down-terminal { };
     emoji-selector = callPackage ../desktops/gnome-3/extensions/emoji-selector { };
+    easyScreenCast = callPackage ../desktops/gnome-3/extensions/EasyScreenCast { };
```
Comment (jtojnar) [inline pkgs/top-level/all-packages.nix:26070]: this should be alphabetical.

File: pkgs/desktops/gnome-3/extensions/EasyScreenCast/default.nix (hunk 1 above)
Comment (doronbehar) [reply pkgs/desktops/gnome-3/extensions/EasyScreenCast/default.nix]: Fixed.

File: pkgs/top-level/all-packages.nix (hunk 2 above)
Comment (doronbehar) [reply pkgs/top-level/all-packages.nix:26069]: I forgot my ABC :).

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104888` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gnomeExtensions.easyScreenCast</li>
  </ul>
</details>

**Comment by doronbehar** [comment]
I will wait a while before merging this, to see it works.

**Comment by doronbehar** [comment]
This is working pretty good - I'm using it for a while now. Upstream isn't responsive regarding the PR there that points to the branch in use here. I'm subscribed to that PR thread, and I'll update the expression if upstream will respond. I'm merging this in the meantime.

**Pull request merged**